src/views/CharacterListView.js

```diff
@@ -11,11 +11,16 @@ class CharacterListView extends React.Component {
 
   componentDidMount() {
     // call our action
+    this.props.getChars();
   }
 
   render() {
-    if (this.props.fetching) {
+    if (this.props.isLoading) {
       // return something here to indicate that you are fetching data
+      return "fetching characters";
+      // {props.isLoading && (
+      //   <Loader type="Ball-Triangle" color="#00BFFF" height="90" width="60" />
+      // )}
     }
```

Comment:
```suggestion
    }
    else if (this.props.error) {
      return <div>{this.props.error}</div>
    }
```
I would add some error handling here